docs/databases/swift-kuery.html

```diff
@@ -21,37 +21,44 @@
   <body>
     <main>
       <h1 class="heading-1">Swift Kuery</h1>
-      <p class="block-text">Swift-Kuery is a pluggable SQL database driver/SDK abstraction layer. Its main idea is to unify the APIs to the various relational databases, providing a Swifty yet SQL-like API.</p>
-      <p class="block-text">This guide uses Swift Kuery with <a onclick="setActiveSidebarElementForParent('raw-routing')" href="../routing/raw-routing.html#">Raw Routing</a> and the <a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL">Swift Kuery PostgreSQL</a> plugin.</p>
+      <p class="block-text"><a href="https://github.com/IBM-Swift/Swift-Kuery" target="_blank">Swift-Kuery</a> is a pluggable SQL database driver/SDK abstraction layer.
+           Its main idea is to unify the APIs to the various relational databases,
+            providing a Swifty yet SQL-like API.
+            This guide will demonstrate how to connect to a SQL Database using one of the Swift-Kuery plugins and how to use this connection to send SQL queries to your database.</p>
       <h2 class="heading-2">Prerequisites</h2>
       <ul class="plain-list">
-        <li><span class="blue-text">Kitura Server:</span> Learn how to create one in our Getting Started guide</li>
-        <li><span class="blue-text">Logging:</span> Learn how to add logging to your project</li>
-        <li><a onclick="localStorage.setItem('package', 'Swift-Kuery'); setActiveSidebarElementForParent('swift-kuery')" href="../getting-started/update-package.html#">Update Dependencies:</a> Swift Kuery needs adding to the Package.swift file.</li>
+          <li><a onclick="setActiveSidebarElementForParent('create-server')" href="../getting-started/create-server.html#">Kitura Server:</a> Learn how to create one in our Getting Started guide.</li>
+          <li><a onclick="setActiveSidebarElementForParent('style-guide')" href="../getting-started/style-guide.html#">Kitura Style Guide:</a> Follow our guide to see how a production ready server should be structured.</li>
+          <li><a onclick="setActiveSidebarElementForParent('style-guide')" href="../getting-started/style-guide.html#">Book Model:</a> You have defined a `Book` struct as described in Step 5 of the style guide.</li>
```

Comment:
I'd prefer -> "Define a Book model, as described in Step 5 of the Style Guide." (we don't want to use "you" anyway)...